File: Android_UI/TelemetrySim/MainActivity.java
package com.example.simm.telemetrysim;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageButton;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ImageButton MyButton = (ImageButton) findViewById(R.id.launch);

        MyButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, Telemetry.class));
            }
        });
    }
}

File: Android_UI/TelemetrySim/MainActivity.cs
using System;
using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;

namespace TelemetrySim
{
    [Activity(Label = "TelemetrySim", MainLauncher = true, Icon = "@drawable/icon")]
    public class MainActivity : Activity
    {
        protected override void OnCreate(Bundle bundle)
        {
            base.OnCreate(bundle);

            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);

            // Get our UI controls from the loaded layout:
            ImageButton launchButton = FindViewById<ImageButton>(Resource.Id.LaunchButton);


            launchButton.Click += (object sender, EventArgs e) =>
            {
                Intent intent = new Intent(this, typeof(TelemetryActivity));
                StartActivity(intent);
            };

        }
    }
}


